stl/inc/execution

```diff
@@ -4189,7 +4189,7 @@ _NODISCARD _Ty transform_reduce(_ExPo&&, _FwdIt1 _First1, _FwdIt1 _Last1, _FwdIt
                         const auto _Chunk1       = _Operation._Basis1._Get_chunk(_Stolen_key);
                         _Val                     = _STD transform_reduce(_Chunk1._First, _Chunk1._Last,
                                                 _Operation._Basis2._Get_first(
-                                                    _Chunk_number, _Operation._Team._Get_chunk_offset(_Chunk_number)),
+                                _Chunk_number, _Operation._Team._Get_chunk_offset(_Chunk_number)),

```

Comment:
The formatting looks wrong. (It looks awful even if correctly formatted, though.)

Reply:
Woof, good catch. I'll see if I can improve this.